cognitedata/reveal
# fix(react-components): allow custom transform on viewer anchor when using stickiness

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/

## Description :pencil:


ViewerAnchor did not properly implement stickiness when content had custom CSS transform. This allows this by going the less elegant route of computing and explicitly setting the CSS transformation in Javascript. 

## How has this been tested? :mag:



In the ViewerAnchor story


## Test instructions :information_source:



- Open ViewerAnchor story
- Move the camera about to put the HTML labels outside the view port, or on its edge
- Observe that the blue one sticks to the edge
- Observe that the blue one also has custom CSS transformation, always placing it above (in screen-space) a point in the corner of the model. 


## Checklist :ballot_box_with_check:
